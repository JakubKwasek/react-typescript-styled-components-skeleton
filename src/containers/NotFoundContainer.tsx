import * as React from "react";
import { Box } from "grommet";

export const NotFoundContainer = (): JSX.Element => (
	<Box
		align="start"
		direction={"row"}
		border={{side: "bottom",}}
		pad="small">
		<h2>404 - Page not found</h2>
		<p>Please enter correct url</p>
	</Box>
);
