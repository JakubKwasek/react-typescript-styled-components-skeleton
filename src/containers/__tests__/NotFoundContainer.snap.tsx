import * as React from "react";
import * as renderer from "react-test-renderer";
import { NotFoundContainer } from "../NotFoundContainer";

describe("NotFoundContainer", () => {
	it("renders correctly", () => {
		const tree = renderer
			.create(<NotFoundContainer />)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
