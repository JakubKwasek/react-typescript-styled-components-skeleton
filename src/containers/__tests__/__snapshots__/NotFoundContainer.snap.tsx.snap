// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundContainer renders correctly 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 bZxUit"
>
  <h2>
    404 - Page not found
  </h2>
  <p>
    Please enter correct url
  </p>
</div>
`;
